crates/cxx-qt-gen/src/naming/name.rs

```diff
@@ -84,6 +84,28 @@ impl Name {
         }
     }
 
+    /// For a given Name generate the Rust and C++ wrapper names
+    pub fn wrapper_from(self) -> Self {
+        const CXX_WRAPPER_SUFFIX: &str = "Wrapper";
+        const RUST_WRAPPER_SUFFIX: &str = "_wrapper";
+
+        let rust_name = self.rust.clone();
+        let cxx = self.cxx.clone();
+
+        let cxx_name = if let Some(name) = cxx {
```

Comment:
Could add the same comment on this as in `from_rust_ident_and_attrs`.

But I would also note that does this get used as if `from_rust_ident_and_attrs` is used to create the Name then it ensures that `.cxx` is not None and where you are using it is using that route.

So is the else case actually required?

Eg if cxx is Some then append the suffix to it, but if cxx is None then only do it on the rust one as the cxx side is not used/set anyway in the original Name so it doesn't have a difference?

Reply:
if cxx is None then it needs to generate a cxx_name anyway it seems from the tests, so I did what the old implementation used, which is that it just converts the rust name to camelcase and uses that to create a wrapper name

Reply:
in other words, if cxx is set to None you still need to generate a cxx wrapper name, just assuming it is the camel-case version of the rust name, not any custom name they provided with cxx_name

Reply:
Right, so guess that works